src/main/groovy/org/labkey/gradle/task/WriteDependenciesFile.groovy

```diff
@@ -22,14 +22,10 @@ import org.gradle.api.artifacts.Configuration
 import org.gradle.api.artifacts.ResolvedArtifact
 import org.gradle.api.tasks.*
 import org.labkey.gradle.plugin.extension.ModuleExtension
-import org.labkey.gradle.util.BuildUtils
 import org.labkey.gradle.util.ExternalDependency
 
 import java.nio.charset.StandardCharsets
 
-// This task can no longer be cacheable since we don't currently declare the jars.txt file as an output file.
-// Once there are no manually maintained jars.txt files, this can be a cacheable task
-//@CacheableTask
 class WriteDependenciesFile extends DefaultTask
```

Comment:
Should this be annotated as a `@CacheableTask` now per the previous comment?

Reply:
I opted not to a this juncture since I'm not convinced it will do the right thing for all version number property changes.

Reply:
Okay